### PR TITLE
Add diamond support

### PR DESCRIPTION
**Pull request by Hackzzila**
Following up on our conversation via email I have added diamond support. I didn't know if you wanted the package to be named `sass-mq` like npm and bower or `mq` as you refer to it in the readme.

To publish the package when you are ready, run the following commands:
Install diamond:
`npm i -g diamondpkg` 

Register for an account:
`diamond login`

Publish the package (run in package dir)
`diamond publish`

### CODE REVIEW

**Comment by kaelig** [comment]
Thanks, that's great! Note that this is an experiment and depending on the success of Diamond (which I want to support), this may get reverted. Let's give it a few months and see how it goes!

Can you post a link to where we can see usage stats for this particular package?

File: diamond.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "sass-mq",
+  "version": "3.3.2",
```
Comment (kaelig) [inline diamond.json]: Could we rely on the version number from package.json instead? I don't want to have to change this value in multiple places.

**Comment by Hackzzila** [comment]
The stats interface is currently being built, it should be done in 1-2 days at most. You bring up a good point, I can support pulling info from package.json files for packages that don't require any of diamond's more advanced features.

**Comment by Hackzzila** [comment]
When using version 0.10.1 it will automatically pull from your `package.json` if no `diamond.json` is found. I have removed `diamond.json`

**Comment by kaelig** [comment]
Sweet! I'll give it a go when it's ready for a spin.

**Comment by Hackzzila** [comment]
If it is diamond you are wondering about, the registry is live right now. I am hoping to deploy download stats on friday. ([Registry status](http://status.diamond.js.org/))

**Comment by kaelig** [comment]
Awesome!

Before merging, I'd like this PR to contain a test. I can do it in the next few weeks (I'm pretty busy right now), but if you have time to write one, I can get this merged sooner!

**Comment by Hackzzila** [comment]
Should the test just be a basic install and compile?

**Comment by kaelig** [comment]
Exactly, this is where tests are run: https://github.com/sass-mq/sass-mq/blob/master/scripts/test.sh

**Comment by Hackzzila** [comment]
I have written some tests.

**Comment by kaelig** [comment]
Thanks @Hackzzila! Let me have a look.

**Comment by Hackzzila** [comment]
Thanks for fixing up my tests! The new site with download counts is near completion.

**Comment by kaelig** [comment]
@Hackzzila looks like the registry is down right now so I'll hold off on merging this until it's fixed and stable, if that's okay!

**Comment by Hackzzila** [comment]
Sorry about that, it was just a bug with the status page. The registry itself seems to be stable. You should be good to go. If you encounter any bugs please let me know!

**Comment by kaelig** [comment]
Hmm, weird, I get this when I try to install a package:

```
diamond install bootstrap
dia ERR! registry error: 404 bootstrap
dia ERR! not ok
```

**Comment by Hackzzila** [comment]
That is because of the move to the new registry. Bootstrap is not currently published to the registry. If you want to install a package from the npm registry, use `diamond install npm:bootstrap`. You could try `diamond install sierra`. The examples are updated in the [new docs](http://vue.diamond.hackzzila.com/#/docs) (note: this is a work in progress).

**Comment by kaelig** [comment]
Thank you! I published sass-mq and it works.

I have two questions:

1. Do we have to run `diamond publish` for every new version or will the registry pick up new tags automatically?
2. Does diamond support dist-tags like npm?

**Comment by Hackzzila** [comment]
1. You currently have to publish for every version, but automation is something to look into.

2. Yes http://vue.diamond.hackzzila.com/#/docs/cli/tag

**Comment by kaelig** [comment]
Got it. Thanks! Ideally publication would be fully automated on this repository and handle both npm and Diamond, then. If you have any pointers / tutorials for doing both, that'd be great, because like a lot of open-source contributors I'm trying to make sure this part is taking as little of my time as possible.

**Pull request merged**